scripts/generators/generate_safe_deployments.py

```diff
@@ -37,14 +40,14 @@ def get_safe_deployments(*args, **kwds):
         clean_resources()
 
 
-def get_network_addresses(safe_deployment: Dict) -> Dict[str, List[str]]:
+def get_network_addresses_by_chain(safe_deployment: Dict) -> Dict[str, List[str]]:
```

Comment:
Typing. Chain should be an int

Reply:
Actually chain is string in the dict, no ints. 
Agree with change to int, but this would be a breaking change for the library. 
Do you want to update it to int in this PR?